test/validators.js

```diff
@@ -1240,7 +1240,7 @@ describe('Validators', () => {
         's!ome.com',
         'domain.com/',
         '/more.com',
-        'domain.com�',
+        'domain.com ',
```

Comment:
This seems like an irrelevant change made by your IDE automatically:
```suggestion
        'domain.com�',
```

Reply:
Right, I use VScode, I'm sure it's that